codeforamerica/vita-min
# Fyst 831/update hoh qp handling

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-831

## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- When a HOH does not have a specified qualified person in the XML, we should move on to the JSON and use that instead.

## How to test?
- Rspec provided
- Navigate through the Arizona flow, selecting Alexis HOH when the opportunity arises.
- At the end of the flow click into 'Main XML Doc'
- See that `<QualChildDependentName>` Has Been filled in correctly.

**Note:** There is currently no persona with _only_ an HoH XML, and valid JSON missing the HoH fields. This cannot therefore be tested for the XML only case for Arizona.

## Screenshots (for visual changes)
<img width="538" alt="image" src="https://github.com/user-attachments/assets/37a85125-c5fa-453e-85bc-1e280ed417c9">
